Pathfinder.elm

```diff
@@ -47,8 +50,12 @@ obstacleTiles obstacles =
   |> List.concat
 
 
-find : (Int, Int) -> List (Int, Int) -> (Int, Int) -> (Int, Int) -> List (Int, Int)
-find = astar
+createMap : List (Int, Int) -> (Int, Int) -> Map
+createMap = Astar.createGrid
+
+
+find : (Int, Int) -> (Int, Int) -> Map -> List (Int, Int)
+find = Astar.findPath
```

Comment:
Sorry! i was wrong, we have some rendering code in `Pathfinder.elm`, so I propose calling it `PathView.elm` with only `render` method. And we can import `Astar` from the `DeliveryPerson`.